.github/workflows/cd.yml

```
@@ -1,7 +1,7 @@
 # See PyPA packaging guide:
 # https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
 
-name: Publish to PyPI
+name: Eradiate delivery
```

For conciseness (see dashboard and graph) I'd prefer to keep the previous one:
```suggestion
name: Publish to PyPI
```